cve_bin_tool/extractor.py

```diff
@@ -67,32 +68,40 @@ def can_extract(self, filename):
     def extract_file_tar(cls, filename, extraction_path):
         """ Extract tar files """
         if not inpath("tar"):
-            print("Error: need 'tar' binary in path")
+            """ Acutally MinGW provides tar, so this might never get called """
+            shutil.unpack_archive(filename, extraction_path)
         else:
             return subprocess.call(["tar", "-C", extraction_path, "-axf", filename])
 
     @classmethod
     def extract_file_rpm(cls, filename, extraction_path):
         """ Extract rpm packages """
-        if not inpath("rpm2cpio") or not inpath("cpio"):
-            print("Error: need 'rpm2cpio' or 'cpio' binary in path")
-        else:
-            with popen_ctx(["rpm2cpio", filename], stdout=subprocess.PIPE) as proc:
-                return subprocess.call(
-                    ["cpio", "-idmv"], stdin=proc.stdout, cwd=extraction_path
+        if sys.platform == "linux":
```

Comment:
the curl rpm test is failing on python2 because sys.platform returns 'linux2' instead of 'linux' on python 2.7.

Reply:
The docs recommend `if sys.platform.startswith('linux'):` (instead of ==) -- apparently in 2.7 era there was talk about distinguishing linux 2.* kernels from linux 3.* kernels, but in the end it turned out that it wasn't super relevant so they switched to just 'linux' from python 3.3 onwards.    https://docs.python.org/2/library/sys.html